sdk/loadtestservice/Azure.Developer.LoadTesting/src/Models/Test.cs

```diff
@@ -0,0 +1,246 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using Azure.Core.Json;
+
+namespace Azure.Developer.LoadTesting.Models
+{
+    /// <summary> Load test model. </summary>
+    public partial class Test
+    {
+        private MutableJsonElement _element;
+
+        /// <summary> Initializes a new instance of Test. </summary>
+        public Test()
```

Comment:
Could you provide an example of required property? I'd like to see how the constructor looks like.

Reply:
There is an int property in LoadTestConfiguration.

I don't think there is a required property in this object graph, but I will add it to my list and provide an example before this work is complete.